app/client/ui/GridViewMenus.ts

```diff
@@ -58,7 +58,7 @@ export function calcFieldsCondition(fields: ViewFieldRec[], condition: (f: ViewF
 }
 
 export function ColumnContextMenu(options: IColumnContextMenu) {
-  const { disableModify, filterOpenFunc, colId, sortSpec, isReadonly, isRaw } = options;
+  const { disableModify, filterOpenFunc, colId, sortSpec, isReadonly } = options;
```

Comment:
Looks very good. The only thing is this logic with `isRaw`. `Hide columns` in MultiColumnMenu should be hidden when `isRaw` is set to true. Can you add this logic there?